src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -165,6 +188,22 @@ void AppleseedRenderer::GetPlatformInformation(MSTR& info) const
 
 #endif
 
+RefTargetHandle	AppleseedRenderer::Clone(RemapDir &remap)
+{
+    AppleseedRenderer* new_rend = static_cast<AppleseedRenderer*>(g_appleseed_renderer_classdesc.Create(false));
+    if (DbgVerify(new_rend != nullptr))
+    {
+        const int num_refs = NumRefs();
+        for (int i = 0; i < num_refs; ++i)
+        {
+            new_rend->ReplaceReference(i, remap.CloneRef(GetReference(i)));
+        }
+        BaseClone(this, new_rend, remap);
+    }
+
+    return new_rend;
+}
+
```

Comment:
I had to implemented Clone method in order to make interactive renderer actually start rendering.